build_all/base-libraries/AzureIoTSocket_Ethernet2/library.properties

```diff
@@ -0,0 +1,9 @@
+name=AzureIoTSocket_Ethernet2
+version=1.0.0
+author=Microsoft
+maintainer=Microsoft <iotcert@microsoft.com>
+sentence=Azure IoT network adapter layer for use with the Adafruit Ethernet FeatherWing on an ESP32 or ESP8266
+paragraph=Microsoft Ethernet2 adaptation layer for connection to an IoT hub. Together with AzureIoTHub, it allows you to use your Arduino with the Azure IoT Hub.  See readme.md for more details.  Copyright (c) Microsoft. All rights reserved.  Licensed under the MIT license. See LICENSE file in the project root for full license information.
+category=Communication
+url=https://github.com/Azure/azure-iot-arduino-socket-ethernet2
+architectures=samd,esp8266,esp32
```

Comment:
Remove esp8266 from architectures. It won't work on that platform.

Reply:
Done.